Source/Core/Core/DSP/DSPCore.h

```diff
@@ -308,6 +308,9 @@ struct SDSP
 
   // This one doesn't really belong here.
   u8* cpu_ram;
+
+  // This is used by DSPJitRegCache to store EBP.
+  u64 ebp_store;
```

Comment:
rbp(like rbx, r12-r15) is callee saved. i'm not sure if our generic entry code saves it along with the others. On x86, ebp could be used by functions as well, provided they saved it. Originally, ebp/rbp was excluded from the list of available registers, and i mechanically added save-code for it, using ebp_store, instead of checking if the generic code handles it.